packages/sdk/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### 0.0.45-alpha.16 -- 2021-03-28
 
+- 2021-04-02 -- Updated `interaction` with metadata object. Added `meta` builder function and test.
```

Comment:
Sorry just noticed this, after the approval, this should be moved up above line 5 (between line 3 and 4) as it isnt a change that is included in the `alpha.16` release of the package. For the record we started with basing things on this changelog stuff: https://keepachangelog.com/en/1.0.0/